frontend/public/components/RBAC/bindings.jsx

```diff
@@ -207,36 +207,44 @@ const roleResources = [
   {kind: 'ClusterRoleBinding', namespaced: false, optional: true},
 ];
 
-export const RoleBindingsPage = ({namespace, showTitle=true, mock}) => <MultiListPage
-  canCreate={true}
-  createButtonText="Create Binding"
-  createProps={{to: '/k8s/cluster/rolebindings/~new'}}
-  mock={mock}
-  filterLabel="by role or subject"
-  flatten={flatten}
-  label="Role Bindings"
-  ListComponent={BindingsList}
-  namespace={namespace}
-  resources={roleResources}
-  rowFilters={[{
-    type: 'role-binding-kind',
-    selected: ['cluster', 'namespace'],
-    reducer: bindingType,
-    items: ({ClusterRoleBinding: data}) => {
-      const items = [
-        {id: 'namespace', title: 'Namespace Role Bindings'},
-        {id: 'system', title: 'System Role Bindings'},
-      ];
-      if (data && data.loaded && !data.loadError) {
-        items.unshift({id: 'cluster', title: 'Cluster-wide Role Bindings'});
-      }
-      return items;
-    },
-  }]}
-  showTitle={showTitle}
-  textFilter="role-binding"
-  title="Role Bindings"
-/>;
+export const RoleBindingsPage = ({namespace, showTitle=true, mock}) => {
+  const createNS = namespace || 'default';
+  const roleBindingAccessReview = {
+    model: RoleBindingModel,
```

Comment:
To be completely accurate, you also need to check if the user has authority to create ClusterRoleBindings